Tasks/PackageDownloader/task.json

```diff
@@ -0,0 +1,110 @@
+{
+  "id": "8d6e8f7e-267d-442d-8c92-1f586864c62f",
+  "name": "PackageDownloader",
+  "friendlyName": "Package Downloader",
+  "description": "Download package from package management.",
+  "helpMarkDown": "",
+  "category": "Utility",      
+  "visibility": [
+    "Build",
+    "Release"
+  ],
+  "author": "ms-vscs-rm",
+  "version": {
+    "Major": "1",
```

Comment:
It's probably a better idea to ship the first version of this task as 0.1.0 until more protocols are supported and we receive feedback from people.